packages/account/PasskeyAccount/passkeyOwnerWebAuthn/background/webAuthn.ts

```diff
@@ -0,0 +1,214 @@
+import { runtime, tabs, windows, type Runtime } from "webextension-polyfill"
```

Comment:
I feel that `passkeyOwnerWebAuthn/{background, content, tabs}` are not so relevant to passkey owner, maybe they could be moved into `packages/webauthn` too.